docs/components/modeler/forms/sidebar-schema.js

```diff
@@ -0,0 +1,29 @@
+const root_dir = "components/modeler/forms/";
+const lib_dir = root_dir + "form-element-library/";
+const config_dir = root_dir + "configuration/";
+
+module.exports = {
+  "Camunda Forms": [
+    root_dir + "camunda-forms-reference",
+    {
+      "Form Element Library": [
+        lib_dir + "forms-element-library",
+        lib_dir + "forms-element-library-text",
+        lib_dir + "forms-element-library-textfield",
+        lib_dir + "forms-element-library-number",
+        lib_dir + "forms-element-library-checkbox",
+        lib_dir + "forms-element-library-radio",
+        lib_dir + "forms-element-library-select",
+        lib_dir + "forms-element-library-checklist",
+        lib_dir + "forms-element-library-taglist",
+        lib_dir + "forms-element-library-button",
+      ],
+    },
+    {
+      Configuration: [
+        config_dir + "forms-config-data-binding",
+        config_dir + "forms-config-options",
+      ],
+    },
+  ],
+};
```

Comment:
👍  this looks like it should work, and the good news is that docusaurus would yell at us during the build if any of these were incorrect.

Non-blocking opinion: converting this list of static IDs into code avoids some repetition, but also adds complexity that I think is unnecessary. I'm not requesting changes, but I do want to state my opinion that this is a pattern I don't think we should follow on future PRs.

Reply:
Thanks for the input here. That's exactly the perspective I was hoping you would offer around complexity and maintenance. I'll keep this in mind and pull you into future PRs, or, ideally, future Define stages where I'm hoping we would have a chance to discuss this with the dev teams before implementation.